v0_5/src/user_stake.rs

```diff
@@ -74,8 +74,8 @@ pub trait UserStakeModule {
             let _ = self.fund_transf_module().swap_unstaked_to_deferred_payment(&unstaked_swappable, || false);
         }
 
-        // swap waiting to active
-        let (affected_users, remaining) = self.fund_transf_module().swap_waiting_to_active(&swappable, || false);
+        // dry run of swap, to get the affected users
+        let (affected_users, remaining) = self.fund_transf_module().swap_waiting_to_active(&swappable.clone(), || false, true);
```

Comment:
i think it would be much cleaner to make a function of self.fund_transf_module().get_possible_affected_users()

Reply:
no need to complicate all the other functions.